experimental/BasisLieHighestWeight/src/WeylPolytope.jl

```diff
@@ -116,7 +116,7 @@ function compute_zero_coordinates(
   zero_coordinates = Int[]
   for c in n:-1:1
     length(non_zeros) == m && break
-    if !isempty(intersect(non_zeros, findall(!iszero, bir_sequence.weights_alpha[c])))
+    if !isdisjoint(non_zeros, findall(!iszero, bir_sequence.weights_alpha[c]))
```

Comment:
This change is not in the scope of the description of the PR, but it makes sense. Did it sneek in from somewhere else and you want to leave it in?

Reply:
there was another similar `isdisjoint` fix that was related to the strings/symbols, and thought I could add this similar change as well (it's from my own experimental folder anyway)

Reply:
fine with me.